x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/risk_score_mapping/index.tsx

```diff
@@ -149,39 +147,29 @@ export const RiskScoreField = ({
   return (
     <EuiFlexGroup direction={'column'}>
       <EuiFlexItem>
-        {
-          // TODO: https://github.com/elastic/kibana/issues/161456
-          // The About step page contains EuiRange component which does not work properly within memoized parents.
-          // EUI team suggested not to memoize EuiRange/EuiDualRange: https://github.com/elastic/eui/issues/6846
-          // Workaround: We force EuiRange re-rendering by removing/adding it into the DOM.
-          // NOTE: We should remove this workaround once EUI team fixed EuiRange.
-          // Related ticket: https://github.com/elastic/kibana/issues/160561
-        }
-        {isActive && (
-          <EuiFormRow
```

Comment:
@e40pud I tested this locally and it appears to be working for me, but would super appreciate a second confirmation/QA! ✨

Reply:
Tested locally. There is no issue anymore! Thanks for fixing this!!